assets/style/app/components/_page-footer.scss

```diff
@@ -73,20 +76,34 @@
 
     .links-social li {
       display: inline-block;
-      margin: 0 $space-base/2;
-      vertical-align: middle;
+      vertical-align: webkit-baseline-middle;
+      margin-right: 15px;
 
       svg {
         margin-top: -2px;
       }
     }
 
     .b-corp-logo {
-      svg > * {
+    svg > * {
         fill: $color-accent-gray;
       }
     }
-
+    .footer-icon-left-bottom > svg {
+      padding-bottom: 8px;
+    }
+    p {
+      line-height: 20px;
+    }
+    #email-subscribtion-form-content {
```

Comment:
hmmm... can you give this form a class name instead of referring to the id in the styling? `id`s are generally reserved for javascript work.